resources/sdk-modules.md

```diff
@@ -0,0 +1,169 @@
+language of choice and write against a set of typed objects native to their
+programming language and environment.
+
+SDKs are often used as a building block to construct a Module.
+
+{% include alert.html type="warning" icon="warning" header="Unsupported"
+body="These SDKs are at an early stage of development and are not supported
+as of yet by Swedbank Pay. They are provided as a convenience to speed up your
+development, so please feel free to play around. However, if you need support,
+please wait for a future, stable release." %}
+
+{:.table .table-striped}
+| Language | SDK  | Repository |
+| :------: | :--- | :--------- |
+{%- for repository in active_repositories -%}
+  {%- if repository.topics contains 'sdk' %}
+| ![{{ repository.language }}](https://img.shields.io/badge/{{ repository.language | downcase | url_encode }}-blue.svg) |
+    {%- if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
+    {%- else -%} {{ repository.description }}
+    {%- endif -%} | [`{{ repository.name }}`]({{ repository.html_url }})
+  {%- endif -%}
+{%- endfor %}
+
+## Unofficial SDKs
+
+{% include alert.html type="warning" icon="warning" header="Unsupported"
+body="Unofficial SDKs are offered as a convenience to speed up your development,
+but are not supported in any way by Swedbank Pay – not now, nor in the future."
+%}
+
+{:.table .table-striped}
+|                         Language                          | Repository                                           | Status                                                                                             |
+| :-------------------------------------------------------: | :--------------------------------------------------- | :------------------------------------------------------------------------------------------------- |
+| ![Node.js](https://img.shields.io/badge/node.js-blue.svg) | [`bjerkio/swedbank-pay-js`][bjerkio-swedbank-pay-js] | [![npm version](https://badge.fury.io/js/swedbank-pay.svg)](https://badge.fury.io/js/swedbank-pay) |
+
+## Official Libraries
+
+**Software libraries** are bundles of code often used by an SDK or in a Module
+to solve one or a set of specific problems.
+
+{% include alert.html type="warning" icon="warning" header="Unsupported"
+body="These libraries are at an early stage of development and are not supported
+as of yet by Swedbank Pay. They are provided as a convenience to speed up your
+development, so please feel free to play around. However, if you need support,
+please wait for a future, stable release." %}
+
+{:.table .table-striped}
+| Language | Library | Repository |
+| :------: | :------ | :--------- |
+{%- for repository in active_repositories -%}
+  {%- if repository.topics contains 'library' %}
+| ![{{ repository.language }}](https://img.shields.io/badge/{{ repository.language | downcase | url_encode }}-blue.svg) |
+    {%- if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)